sos/report/__init__.py

```diff
@@ -904,6 +911,12 @@ def _set_all_options(self):
                         opt.value = True
 
     def _set_tunables(self):
+        # Set plugin option's defaults to the global argument values
+        for pluginname, plugin in self.loaded_plugins:
+            for optname in plugin.options:
+                if hasattr(self.opts, optname):
+                    plugin.options[optname].value = getattr(self.opts, optname)
```

Comment:
I'm not seeing why this is needed? We have the defaults for the plugin options set when they're instantiated as a `PluginOpt`, and these shouldn't differ at this point.

Reply:
Nevermind, I see it now. The other options to date have had specific handling to them for default vs. plugin-specific values due to the need to do some dancing with magic numbers. We don't need to do that here, so we're good.